src/components/BookstoreContainer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+
+import Navbar from './Navbar';
+import BooksList from './BooksList';
+import Categories from './Categories';
+
+function BookstoreContainer() {
+  return (
+    <div className="bookstore-container">
+      <Router>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<BooksList />} />
+          <Route path="/categories" element={<Categories />} />
+        </Routes>
```

Comment:
Has 2 `<Route>`s, one for the BookList component and the other for the Categories component 🙌 Nice!